canonical/ubuntu.com
# Update Third party and geolocation policy

## QA

- Check https://ubuntu-com-13700.demos.haus/legal/data-privacy#where-we-transfer-and-store-your-information has the [updated pdf ](https://docs.google.com/document/d/1qesV0RGWxv1lkK8K1hEprwQqTpiIcLhluJVFdT__iLE/edit)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-9964


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
